projects/igniteui-angular/src/test.ts

```diff
@@ -15,20 +15,24 @@ import localeFr from '@angular/common/locales/fr';
 import localeDe from '@angular/common/locales/de';
 import localeJa from '@angular/common/locales/ja';
 
-declare const require: any;
+declare const require: {
+  context(path: string, deep?: boolean, filter?: RegExp): {
+    <T>(id: string): T;
+    keys(): string[];
+  };
+};
 
 registerLocaleData(localeFr);
 registerLocaleData(localeDe);
 registerLocaleData(localeJa);
 
 // First, initialize the Angular testing environment.
-// First, initialize the Angular testing environment.
-getTestBed().initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting(), {
-    teardown: { destroyAfterEach: false }
-});
+getTestBed().initTestEnvironment(
+  BrowserDynamicTestingModule,
+  platformBrowserDynamicTesting(),
+);
+
 // Then we find all the tests.
 const context = require.context('./', true, /\.spec\.ts$/);
 // And load the modules.
 context.keys().map(context);
-
-jasmine.DEFAULT_TIMEOUT_INTERVAL = 15000;
```

Comment:
Might consider restoring this or at least increasing the default 5000ms if some tests are still regularly hitting the timeout limit. So far at least one or two flaked out like that, which isn't too promising, though I'm thinking those tests could be looked at, because they shouldn't take this long anyway